### ISSUE TITLE
Test not tolerant of whitespace differences.

### ISSUE THREAD

**Issue by BlauFeuer** [FreeCodeCamp/FreeCodeCamp#10324]
Challenge [Size your Images](https://www.freecodecamp.com/challenges/size-your-images#?solution=%0A%3Clink%20href%3D%22https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DLobster%22%20rel%3D%22stylesheet%22%20type%3D%22text%2Fcss%22%3E%0A%3Cstyle%3E%0A%20%20.red-text%20%7B%0A%20%20%20%20color%3A%20red%3B%0A%20%20%7D%0A%0A%20%20h2%20%7B%0A%20%20%20%20font-family%3A%20Lobster%2C%20Monospace%3B%0A%20%20%7D%0A%20%20.smaller-image%3A%20%7Bwidth%3A%20100px%7D%0A%20%20p%20%7B%0A%20%20%20%20font-size%3A%2016px%3B%0A%20%20%20%20font-family%3A%20Monospace%3B%0A%20%20%7D%0A%3C%2Fstyle%3E%0A%0A%3Ch2%20class%3D%22red-text%22%3ECatPhotoApp%3C%2Fh2%3E%0A%0A%3Cimg%20class%3D%27smaller-image%27%20src%3D%22https%3A%2F%2Fbit.ly%2Ffcc-relaxing-cat%22%20alt%3D%22A%20cute%20orange%20cat%20lying%20on%20its%20back.%20%22%3E%0A%0A%3Cp%20class%3D%22red-text%22%3EKitty%20ipsum%20dolor%20sit%20amet%2C%20shed%20everywhere%20shed%20everywhere%20stretching%20attack%20your%20ankles%20chase%20the%20red%20dot%2C%20hairball%20run%20catnip%20eat%20the%20grass%20sniff.%3C%2Fp%3E%0A%3Cp%20class%3D%22red-text%22%3EPurr%20jump%20eat%20the%20grass%20rip%20the%20couch%20scratched%20sunbathe%2C%20shed%20everywhere%20rip%20the%20couch%20sleep%20in%20the%20sink%20fluffy%20fur%20catnip%20scratched.%3C%2Fp%3E%0A) has an issue of not being tolerant of whitespace differences.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:51.0) Gecko/20100101 Firefox/51.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  .red-text {
    color: red;
  }

  h2 {
    font-family: Lobster, Monospace;
  }
  .smaller-image: {width: 100px}
  p {
    font-size: 16px;
    font-family: Monospace;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<img class='smaller-image' src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back. ">

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>

```
fails with: ✘ **Your image should be 100 pixels wide. Browser zoom should be at 100%.**

**Comment by erictleung** [comment]
@BlauFeuer thanks for the issue. This case, however, your code is incorrect. Please double check that your CSS rules are formatted correctly. Happy coding!

**Comment by BlauFeuer** [comment]
The formatting via whitespace seems to be the cause of the challenge failure.
I believe the CSS rules are formatted correctly per https://www.w3.org/TR/REC-CSS2/syndata.html
Specifically 
```css
.smaller-image: {width: 100px}
```

> re:  A declaration is either empty or consists of a property, followed by a colon (:), followed by a value. Around each of these there _may be whitespace._
> 
> Because of the way selectors work, multiple declarations for the same selector _may_ be organized into semicolon (;) separated groups.

**Comment by BlauFeuer** [comment]
@erictleung  - Yes, the code above won't pass.  Thanks for the help and well-wishes.  I just wanted to explain the bigger picture problem I was having all day long in case this ever happens to someone else:

My code would fail even with good code. Since the test check the image display size, if the code is correct but an ad-blocker prevents that image from loading, it will always fail - which lead me to keep trying _wrong_ things to see if I could see any pattern to it. Once the ad-blocker was off, I passed the challenge immediately.  I just started here at Free Code Camp and I am looking forward to being less of a code klutz.  Again, thanks for the encouragement.
